package.json

```diff
@@ -17,52 +17,53 @@
   },
   "scripts": {
     "build": "ember build",
+    "lint:hbs": "ember-template-lint .",
     "lint:js": "eslint .",
     "start": "ember serve",
     "test": "ember test",
     "test:all": "ember try:each"
   },
   "dependencies": {
-    "ember-cli-babel": "^6.14.1"
+    "ember-cli-babel": "^7.7.3"
   },
   "devDependencies": {
-    "@ember-decorators/babel-transforms": "^2.0.1",
-    "broccoli-asset-rev": "^2.7.0",
-    "ember-cli": "~3.1.3",
-    "ember-cli-addon-docs": "elwayman02/ember-cli-addon-docs#hero-yield",
+    "@ember/optional-features": "^0.7.0",
+    "broccoli-asset-rev": "^3.0.0",
+    "ember-cli": "~3.12.0",
+    "ember-cli-addon-docs": "^0.6.14",
     "ember-cli-addon-docs-yuidoc": "^0.2.1",
     "ember-cli-app-version": "^3.2.0",
-    "ember-cli-dependency-checker": "^2.1.1",
+    "ember-cli-dependency-checker": "^3.1.0",
     "ember-cli-deploy": "^1.0.2",
     "ember-cli-deploy-build": "^1.1.1",
     "ember-cli-deploy-git": "^1.3.3",
     "ember-cli-deploy-git-ci": "^1.0.1",
-    "ember-cli-eslint": "^4.2.3",
-    "ember-cli-htmlbars": "^2.0.3",
-    "ember-cli-htmlbars-inline-precompile": "^1.0.3",
-    "ember-cli-inject-live-reload": "^1.7.0",
-    "ember-cli-qunit": "^4.3.2",
+    "ember-cli-eslint": "^5.1.0",
+    "ember-cli-htmlbars": "^3.0.1",
+    "ember-cli-htmlbars-inline-precompile": "^2.1.0",
+    "ember-cli-inject-live-reload": "^2.0.1",
     "ember-cli-release": "1.0.0-beta.2",
-    "ember-cli-shims": "^1.2.0",
     "ember-cli-sri": "^2.1.1",
-    "ember-cli-uglify": "^2.1.0",
+    "ember-cli-template-lint": "^1.0.0-beta.1",
+    "ember-cli-uglify": "^3.0.0",
     "ember-disable-prototype-extensions": "^1.1.3",
     "ember-export-application-global": "^2.0.0",
-    "ember-load-initializers": "^1.1.0",
+    "ember-load-initializers": "^2.0.0",
     "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-resolver": "^4.5.6",
-    "ember-sinon": "2.1.0",
-    "ember-sinon-qunit": "3.1.0",
-    "ember-source": "~3.2.2",
-    "ember-source-channel-url": "^1.1.0",
-    "ember-try": "^0.2.23",
-    "eslint-plugin-ember": "^5.2.0",
-    "eslint-plugin-node": "^6.0.1",
+    "ember-qunit": "^4.4.1",
+    "ember-resolver": "^5.0.1",
+    "ember-sinon": "^4.0.0",
+    "ember-sinon-qunit": "^3.4.0",
+    "ember-source": "~3.12.0",
+    "ember-source-channel-url": "^2.0.1",
+    "ember-try": "^1.0.0",
+    "eslint-plugin-ember": "^7.0.0",
+    "eslint-plugin-node": "^9.0.1",
     "loader.js": "^4.7.0",
-    "qunit-dom": "^0.7.0"
+    "qunit-dom": "^0.9.0"
   },
   "engines": {
-    "node": "6.* || 8.* || >= 10.*"
+    "node": "8.* || >= 10.*"
```

Comment:
@elwayman02 @tylerturdenpants when releasing this please keep in mind that this is a breaking change :)

also, do you have time to put out a new release including this change? would very much like to use Babel 7 in our app 😅